_posts/en/newsletters/2024-02-14-newsletter.md

```diff
@@ -0,0 +1,295 @@
+  a0link="https://bitcoincore.reviews/28950#l-27"
+
+  q1="Are there other important checks apart from `maxburnamount` and
+      `maxfeerate` that should be performed on packages before they are
+      accepted to the mempool?"
+  a1="Yes, two examples are the base fee check and maximum standard
+      transaction size. These are low-cost checks, so can be checked
+      early and fail the package quickly."
+  a1link="https://bitcoincore.reviews/28950#l-33"
+
+  q2="The options `maxburnamount` and `maxfeerate` can prevent a
+      transaction from entering the mempool and being relayed. Can we
+      consider these options as policy rules? Why or why not?"
+  a2="This is policy; these checks don't apply to transactions in
+      mined blocks (so this isn't consensus). These don't even
+      affect relay of transactions from peers, only transactions
+      submitted locally using RPC."
+  a2link="https://bitcoincore.reviews/28950#l-47"
+
+  q3="Why do we validate maxfeerate against the modified feerate
+      instead of the base fee rate?"
+  a3="(Earlier review clubs [24152][review club 24152],
+      [24538][review club 24538], and [27501][review club 27501]
+      covered the concept of modified versus base fees.)
+      Most participants thought the base fee should be used instead
+      of the modified fee, because `sendrawtransaction` and `testmempoolaccept`
+      use the base fee in their checks, so that would seem more consistent.
+      It may not make any practical difference since `prioritisetransaction`
+      (which makes modified and base fees different) is generally used
+      only by miners."
+  a3link="https://bitcoincore.reviews/28950#l-69"
+
+  q4="We validate maxfeerate against the modified feerate of individual
+      package transactions, not package feerate. When can this be inaccurate?"
+  a4="When a package child transaction is rejected because its modified
+      fee rate exceeds `maxfeerate` individually, but does not if it's checked
+      as a package."
+  a4link="https://bitcoincore.reviews/28950#l-84"
+
+  q5="Given that possible inaccuracy, why not check `maxfeerate` against
+      package feerate instead?"
+  a5="Because this can cause a different inaccuracy. Suppose
+      transaction A has zero fee and B CPFPs (bumps) A. Both A and B
+      are physically large so neither exceeds `maxfeerate`. But now
+      high-fee rate C is added which spends from both A and B.
+      (This is an allowed package topology because it's only two levels,
+      although it was pointed out that the `submitpackage` RPC doesn't allow
+      this topology.) In this case, C would be accepted because much of its
+      fee is absorbed by A and B, but C should be rejected."
+  a5link="https://bitcoincore.reviews/28950#l-108"
+
+  q6="Why can’t `maxfeerate` be checked immediately after decoding like
+      `maxburnamount` is?"
+  a6="Because transaction inputs famously don't explicitly state the
```

Comment:
famously 🤣

Reply:
Maybe I shouldn't have tried to be cute there, it just kind of slipped out, wasn't my fault! :grin: